mattermost.go

```diff
@@ -530,13 +545,19 @@ func convertTeamTo30(primaryTeamName string, team *TeamForUpgrade, uniqueEmails
 			}
 		}
 
+		if user.AuthData != nil && *user.AuthData != "" && uniqueAuths[*user.AuthData] {
+			shouldUpdateUser = true
+		}
+
 		if shouldUpdateUser {
 			if _, err := store.GetMaster().Exec(`
 				UPDATE Users 
 				SET 
 				    Email = :Email,
 				    Username = :Username,
-				    Roles = :Roles
+				    Roles = :Roles,
+				    AuthService = '',
```

Comment:
I don't have this very clear, should the update User set AuthService to what he had and AuthData to match the new email address?